project.json

```diff
@@ -14,6 +14,7 @@
     "icon_path": "preview.png",
     "engine": "o3de",
     "external_subdirectories": [
+        "MPSGameLift",
         "Gem"
```

Comment:
Can you swap these lines around to keep them in alphabetical order?

Reply:
done